packages/core/storybook/Step.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import Step from '../src/Step'
+
+const onClick = () => alert('Step Clicked')
+
+storiesOf('Step', module)
+  .add(
+    'Step component',
+    withInfo({
+      inline: true,
+      text: 'Use the <Step /> component to render a step.'
```

Comment:
Stepper.Step?

Reply:
That's fair, I should keep it consistent 👍